### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by WahidIslamLinad** [THEREALWWEFAN231/Inertia#1090]
**Minecraft Version(s)**
1.16.4
**What is the suggestion**
Adding Seed Cracker & antixray bypass
**Why would this feature be useful**
Now a days there are so many server's with antixray which is a pain..but there are ways to bypass them if we can crack the seed..So for bypassing antixray & also seed cracking i think the dev should add Seed cracking & Antixray bypass..
i found a antixray bypass mod on github..link is down below
https://github.com/19MisterX98/DiamondSim
and i guess the same dev also has a seed cracker or you can use the one neil made..
**Final checklist:**

- [ ] I know how to properly use check boxes 
- [ ] This is not a suggestion for baritone or other mod

**Comment by feliscatusmeows** [comment]
lol no that would be too bloated

**Comment by constantinDev** [comment]
an actual good antixray bypass: https://github.com/AriliusClient/AntiAntiXray
seed cracker isnt needed for that one

**Comment by feliscatusmeows** [comment]
yes also apparently ore generation varies depending on the order of chunks, also the server can use different ore generation so like don't use seeds for aaxray

**Comment by WahidIslamLinad** [comment]
> an actual good antixray bypass: https://github.com/AriliusClient/AntiAntiXray
> seed cracker isnt needed for that one

@constantinDev is it for 1.16.4 or 5 ??

**Comment by constantinDev** [comment]
> > an actual good antixray bypass: https://github.com/AriliusClient/AntiAntiXray
> > seed cracker isnt needed for that one
> 
> @constantinDev is it for 1.16.4 or 5 ??

works on almost any version, means any version fabric supports

**Comment by WahidIslamLinad** [comment]
@constantinDev hello again..can you provide the download link? its says page not found

**Comment by constantinDev** [comment]
> @constantinDev hello again..can you provide the download link? its says page not found

forgot to update it, latest download is in build/libs

**Comment by WahidIslamLinad** [comment]
> forgot to update it, latest download is in build/libs
@constantinDev thnx for the mod..but ig this wont work for my server..i tried so man times but there's always some leftover blocks..tho better thn noting ig <3

**Comment by constantinDev** [comment]
> > forgot to update it, latest download is in build/libs
> > @constantinDev thnx for the mod..but ig this wont work for my server..i tried so man times but there's always some leftover blocks..tho better thn noting ig <3

Can i get your server so i can test it myself?

**Comment by WahidIslamLinad** [comment]
> > > forgot to update it, latest download is in build/libs
> > > @constantinDev thnx for the mod..but ig this wont work for my server..i tried so man times but there's always some leftover blocks..tho better thn noting ig <3
> 
> Can i get your server so i can test it myself?

its a cracked server but only ip from my country is allowed..if you want to see ig i can show you on discord if you have..
